appveyor.yml

```diff
@@ -66,24 +66,18 @@ test_script:
 
       if ($testsExitCode -ne 0) {exit $testsExitCode}
 
-  - ps: |
-      & coverage erase
+  - coverage erase --rcfile=.coveragerc_appveyor
+  - del /Q coverage.xml
 
-      $clcachePy = "$((Get-Location).Path)\clcache.py"
-      $env:CLCACHE_CMD = "coverage run -a --branch --include=$clcachePy $clcachePy";
-      & py.test --junitxml .\integrationtests.xml integrationtests.py --cov=clcache
+  - ps: |
+      & py.test --junitxml .\integrationtests.xml integrationtests.py --cov=clcache --cov-config=.coveragerc_appveyor
```

Comment:
This part looks odd: does this mean that during the integration tests, the individual `clcache.py` processes are no longer launched via `coverage`? If so, how does it work that coverage information is collected? Does the plugin for `py.test` magically hook into subprocesses?